java/apply-scheduled-updates-to-preplanned-map-area/README.md

```diff
@@ -54,8 +54,5 @@ The data in this sample shows the roads and trails in the Canyonlands National P
 **Note:** preplanned areas using the Scheduled Updates workflow are read-only. For preplanned areas that can be edited on the end-user device, see [Take a map offline - preplanned](https://developers.arcgis.com/java/latest/guide/take-map-offline-preplanned.htm).
 
 #### Tags
-Edit and Manage Data
-offline, preplanned
-pre-planned
-synchronize
-update
+
+offline, pre-planned, preplanned, synchronize, update
```

Comment:
I can't comment on this file because most of it wasn't changed, but:

1. The image description is in the old format.
2. The offline data still has sdcard refs in there.

Reply:
will be handled in a follow up PR